src/uct/api/uct.h

```diff
@@ -149,6 +149,37 @@ typedef struct uct_tl_resource_desc {
 #define UCT_TL_RESOURCE_DESC_ARG(_resource)   (_resource)->tl_name, (_resource)->dev_name
 
 
+/**
+ * @brief Atomic operation requested for uct_ep_atomic_post
+ *
+ * This enumeration defines which atomic memory operation should be
+ * performed by the uct_ep_atomic_post family of fuctions. All of these are
+ * non-fetching atomics and will not result in a request handle.
+ */
+typedef enum uct_atomic_post_op {
+    UCT_ATOMIC_POST_OP_ADD,
+    UCT_ATOMIC_POST_OP_AND,
+    UCT_ATOMIC_POST_OP_OR,
+    UCT_ATOMIC_POST_OP_XOR,
+    UCT_ATOMIC_POST_OP_LAST
+} uct_atomic_post_op_t;
+
+/**
+ * @brief Atomic operation requested for uct_ep_atomic_fetch_nb
+ *
+ * This enumeration defines which atomic memory operation should be
+ * performed by the uct_ep_atomic_fetch_nb family of fuctions. All
+ * of these functions will fetch data from the remote node.
+ */
+typedef enum uct_atomic_fetch_op {
+    UCT_ATOMIC_FETCH_OP_ADD  = UCT_ATOMIC_POST_OP_ADD,
```

Comment:
pls document every member of the enum, also i see no reason to assign FETCH ops the values from POST ops

Reply:
documented, fetch-op codes are out of sync